repositorytools/cli/common.py

```diff
@@ -37,12 +37,15 @@ def _get_parser(self):
 
     def __init__(self):
         self.parser = self._get_parser()
-        self.parser.add_argument("-D", "--debug", action="store_true", dest="debug", default=False,
-                                 help="Print lots of debugging information")
+        self.parser.add_argument("-D", "--debug", action="store_true", help="Print lots of debugging information")
+        self.parser.add_argument("-V", action="store_true", dest="display_version", help="Prints version and exit")
         self.repository = None
 
     def run(self, args=None):
         args_namespace = self.parser.parse_args(args)
+        if args_namespace.display_version:
```

Comment:
This functionality didn't work for me in Python 2.6 or 2.7 for some reason. It exited, but didn't print anything.

 Please try a solution working also in python >= 2.6